packages/data-wizard/__tests__/dataset/data-frame.test.ts

```diff
@@ -358,19 +383,15 @@ describe('DataFrame Get Value Functions', () => {
   });
 });
 
-// describe('New DataFrame', () => {
+// describe('DataFrame Info', () => {
```

Comment:
UCC

Reply:
In the next PR, DW will allow comparison of two Series or two DF directly, and I will fix it.